specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2019-05-01/machineLearningServices.json

```diff
@@ -1620,6 +2077,166 @@
       ],
       "x-ms-discriminator-value": "AmlCompute"
     },
+    "AmlSpark": {
+      "description": "An Azure Machine Learning Spark compute.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Compute"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "properties": {
+              "description": "AML Spark Compute properties",
+              "type": "object",
+              "properties": {
+                "sparkVersion": {

```

Comment:
Is this finite? Should it be an enum or is it open?

Reply:
No, this list keeps growing and we don't want the swagger/SDK update on new spark version support.. That's why we modelled as string.